plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -5,6 +5,7 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
+import urllib
```

Comment:
```suggestion
import urllib
```
use `fetch_url` or `open_url` from `module_utils/urls.py`